.tugboat/config.yml

```diff
@@ -36,33 +44,69 @@ services:
                 # Install the PHP extensions
                 - docker-php-ext-install mysqli exif zip
 
-                # Install wp-cli
-                - curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
-                - chmod +x wp-cli.phar
-                - mv wp-cli.phar /usr/local/bin/wp
-
                 # Install WordPress.
                 - composer install --optimize-autoloader
 
                 # OPTION 2: WordPress Core lives in a subdirectory (in this example, 'docroot').
                 - ln -snf "${TUGBOAT_ROOT}/docroot" "${DOCROOT}"
 
                 # Set the wp-config.php and .htaccess files with the one you defined for Tugboat.
-                - rm -rf ${DOCROOT}/wp-config.php
-                - cp ${TUGBOAT_ROOT}/.tugboat/wp-config.tugboat.php ${DOCROOT}/wp-config.php
+                - cp ${TUGBOAT_ROOT}/.tugboat/wp-config.tugboat.php ${DOCROOT}/wp-config.local.php
                 - cp ${TUGBOAT_ROOT}/.tugboat/.htaccess ${DOCROOT}/.htaccess
 
-                # Update permalinks to remove the index.php.
-                - wp --allow-root --path="${DOCROOT}" option set permalink_structure /%postname%/
-                - wp --allow-root --path="${DOCROOT}" rewrite flush --hard
 
             # Phase 2 (update): Import files, database, or any other assets that your
             # website needs to run.
             # When you refresh a Tugboat Preview, the process starts here, skipping `init`.
             update:
-                # Import uploads
-                - tar -C /tmp -zxf uploads.tar.gz
-                - rsync -avz --delete /tmp/uploads/ "${TUGBOAT_ROOT}/docroot/wp-content/uploads/"
+                # Ensure our private SSH key is in place so that we can connect to remote services.
+                # Note: This assumes you have added your private key as a Tugboat environment variable for this project.
+                #       In this example, that environment variable is named "MY_PRODUCTION_SSH_KEY"
+                - echo "${MY_PRODUCTION_SSH_KEY}" > ~/.ssh/production_ssh_key
+                - chmod 600 ~/.ssh/production_ssh_key
```

Comment:
Let's test this and move it to the Tutorial if it works.  For the demo we want to keep things simple and include the files and database in the repo.

Reply:
Sounds good. This won't be needed when we add the database and files back into the repo.